NinchatSDKSwift/Implementations/Data Source - Delegate/Questionnaire/QuestionnaireDataSourceDelegate.swift

```diff
@@ -214,7 +199,7 @@ extension QuestionnaireDataSourceDelegate {
             {
                 "name": "register-logic",
                 "logic": {
-                  "target": "_register"
+                  "target": "_exit"
```

Comment:
What is the purpose for this? There is no "_exit" target supported in site configs. Only _register or _complete

Reply:
This is the target for the _audienceRegisteredText_ and _audienceCompletedText_ elements. To avoid double registering, I have to use another target.